cdp_scrapers/legistar_utils.py

```diff
@@ -747,7 +749,14 @@ def get_roles(
             [
                 self.get_none_if_empty(
                     Role(
-                        body=self.get_body(record[LEGISTAR_ROLE_BODY]),
+                        body=(
+                            self.get_body(record[LEGISTAR_ROLE_BODY])
+                            or self.get_none_if_empty(
+                                Body(
+                                    name=str_simplified(record[LEGISTAR_ROLE_BODY_ALT]),
+                                )
+                            )
+                        ),
```

Comment:
I'm confused. LEGISTAR_ROLE_BODY = "OfficeRecordBodyInfo" doesn't exist?

Reply:
When we query and retrieve OfficeRecords, we ourselves add a OffceRedcordBodyInfo per OfficeRecord using OfficeRecordBodyId

https://github.com/CouncilDataProject/cdp-scrapers/blob/868bb64c3f18da1eb22781c2a4feb16ef3efd3d3/cdp_scrapers/legistar_utils.py#L225-L230

So record["OfficeRecordBodyInfo"] = response JSON for body = OfficeRecordBodyId

Reply:
Apparently in some cases, the query for Body fails, because they entered incorrect OfficeRecordBodyId. I just found out myself when checking results for this PR code.

i.e. I saw we got `Roles` with nonempty `title` but empty `Body`.

Reply:
Nevermind, I got it